### PR TITLE
update masking and warning for old models

### PR DESCRIPTION
**Pull request by nithinraok**
Signed-off-by: nithinraok <nithinrao.koluguri@gmail.com>

### CODE REVIEW

File: nemo/collections/asr/models/label_models.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def forward(self, input_signal, input_signal_length):
         )
 
         encoded, length = self.encoder(audio_signal=processed_signal, length=processed_signal_len)
-        logits, embs = self.decoder(encoder_output=encoded, length=length)
+        logits, embs = self.decoder(encoder_output=encoded, length=None)
```
Comment (tango4j) [inline nemo/collections/asr/models/label_models.py:196]: Is there any other use of length variable for decoder?
If there is an expected future use, you may leave it
otherwise setting it with default length=None would make it neater.

**Review by tango4j — COMMENTED** [review]
A comment was added to self.decoder() part

**Review by tango4j — APPROVED** [review]
